src/Organizer/Manager/GeoCoordinatesProcessManager.php

```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Organizer\Manager;
```

Comment:
Nit: If you put this `ProcessManager` in a dedicated namespace, `CultuurNet\UDB3\Organizer\ProcessManager` might be better than `Manager`. Process Managers are a specific term used in CQRS, it's not some kind of Manager we made up: https://medium.com/@drozzy/long-running-processes-event-sourcing-cqrs-c87fbb2ca644